contracts/kleros/KlerosGovernor.sol

```diff
@@ -104,16 +104,16 @@ contract KlerosGovernor is Arbitrable {
      *  @param _description The string in CSV format that contains labels of list's transactions.
      *  Note that the submitter may give bad descriptions of correct actions, but this is to be seen as UI enhancement, not a critical feature and that would play against him in case of dispute.
      */
-    event ListSubmitted(uint indexed _listID, address indexed _submitter, uint _session, string _description);
+    event ListSubmitted(uint indexed _listID, address indexed _submitter, uint indexed _session, string _description);
 
     /** @dev Constructor.
-     *  @param _arbitrator The arbitrator of the contract. It should support appealPeriod.
+     *  @param _arbitrator The arbitrator of the contract. It must support appealPeriod.
      *  @param _extraData Extra data for the arbitrator.
      *  @param _submissionBaseDeposit The base deposit required for submission.
      *  @param _submissionTimeout Time in seconds allocated for submitting transaction list.
      *  @param _executionTimeout Time in seconds after approval that allows to execute transactions of the approved list.
      *  @param _withdrawTimeout Time in seconds after submission that allows to withdraw submitted list.
-     *  @param _sharedMultiplier Multiplier of the appeal cost that submitters has to pay for a round when there is no winner/loser in the previous round. In basis points.
+     *  @param _sharedMultiplier Multiplier of the appeal cost that submitters has to pay for a round when there is no winner/loser in the previous round (e.g. when there isn't a previous round). In basis points.
```

Comment:
I think this wording makes it more confusing. I'd rather say "e.g when it's the very first round"